tox.ini

```diff
@@ -0,0 +1,26 @@
+;
```

Comment:
Yes. I will move it to sdks/python. Same location as setup.py.

For Jenkins comment, I will work on it after this change.